src/Microsoft.AspNet.Mvc.TagHelpers/LinkTagHelper.cs

```diff
@@ -183,7 +200,11 @@ public override void Process(TagHelperContext context, TagHelperOutput output)
 
             var builder = new DefaultTagHelperContent();
 
-            if (mode == Mode.Fallback && string.IsNullOrEmpty(HrefInclude))
+            if (mode == Mode.FileVersion)
+            {
+                BuildLinkTag(attributes, builder);
+            }
+            else if (mode == Mode.Fallback && string.IsNullOrEmpty(HrefInclude))
```

Comment:
Hmmm, I thought this was more readable since we have different sections for different modes..